src/Features/Core/Diagnostics/AnalyzerHelper.cs

```diff
@@ -1,9 +1,11 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.ErrorReporting;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Diagnostics
```

Comment:
My latest review removes a lot of methods that I recently added from this helper type. It will now just have 2 static methods used by IDE driver and HostAnalyzerManager for supported diagnostics.